src/superphot_plus/data_generation/make_fake_spp_data.py

```diff
@@ -116,42 +116,79 @@ def create_prior(cube, tdata):
 
     return cube
 
+def ztf_noise_model(mag, band, snr_range_g = [1,10], snr_range_r=[1,10]):
+    """
+    A very, very simple noise model which assumes the dimmest magnitude is at SNR = 1, and the brightest mad is at SNR = 10
+
+    inputs: 
+    mag: observed magnitude
+    band: observed band (g or r)
+    snr_range_g: Range of signal-to-noise ratios you want to see in g-band
+    snr_range_r: Range of signal-to-noise ratios you want to see in g-band
 
-def create_clean_models(nmodels, num_times=100):
```

Comment:
Removing this function will break the unit tests in tests/superphot_plus/data_generation/test_make_fake_spp_data.py. One solution is to remove those tests. However, I actually think it is useful to have a function that generates clean models to use in testing. Is that possible to do with the new function (using the snr ranges)?

Reply:
Thank you!